### PR TITLE
[azingest] add sovereign support

### PR DESCRIPTION
**Pull request by gracewilcox**
Adding sovereign cloud support for Monitor Ingestion.

Fixes https://github.com/Azure/azure-sdk-for-go/issues/21648

### CODE REVIEW

File: sdk/monitor/azingest/cloud_config.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azingest
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+// Cloud Service Names for Monitor Query Logs and Metrics, used to identify the respective cloud.ServiceConfiguration
```
[the remark this answers is not in the document]
Comment (gracewilcox) [reply sdk/monitor/azingest/cloud_config.go]: Thanks! Fixed in 38340da

File: sdk/monitor/azingest/README.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ The [azidentity][azure_identity] module is used for Azure Active Directory authe
 
 An authenticated client object is required to upload logs. The examples demonstrate using [azidentity.NewDefaultAzureCredential][default_cred_ref] to authenticate; however, the client accepts any [azidentity][azure_identity] credential. See the [azidentity][azure_identity] documentation for more information about other credential types.
 
+The clients default to the Azure public cloud. For other cloud configurations, see the [cloud][cloud_documentation] package documentation.
```
Comment (scottaddie) [inline sdk/monitor/azingest/README.md:34]:
```suggestion
The client defaults to the Azure public cloud. For other cloud configurations, see the [cloud][cloud_documentation] package documentation.
```

**Review by scottaddie — APPROVED** [review]

File: sdk/monitor/azingest/README.md (hunk 2 above)
Comment (gracewilcox) [reply sdk/monitor/azingest/README.md]: Fixed in 2f00690

**Pull request merged**